.travis.yml

```diff
@@ -10,6 +10,7 @@ addons:
 before_install:
 - git config --global user.email "algobot@users.noreply.github.com"
 - git config --global user.name "algobot"
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.10.1
```

Comment:
Those are not indented correctly (so are the two previous lines for some reason).